src/index.css

```diff
@@ -5,4 +5,9 @@ html {
 
 body {
     margin: 0;
+    height: 100%;
 }
+
+#root {
+    height: 100%;
+}
```

Comment:
?

Reply:
It is needed for positioning and styling.

Reply:
the root div had a fixed height, so it wasn’t possible to center vertically